executor/executor.go

```diff
@@ -693,7 +693,7 @@ func init() {
 type ProjectionExec struct {
 	baseExecutor
 
-	exprs            []expression.Expression // Only used in Next().
+	exprs            []expression.Expression
```

Comment:
As comments suggests, this variable is only used in Next(). Since you are trying to remove Next(), you probably need also remove this. ``